apps/checker/app/services/MatcherPool.scala

```diff
@@ -375,7 +375,7 @@ class MatcherPool(
     * available categories.
```

Comment:
This comment needs to be changed to reflect how this now works.

Reply:
I think this is how it works as of this PR - if I ask for an empty list of categories, it defaults to all.

Previously, if I asked for an empty list of categories, it would give me no categories.

Reply:
I think the person who wrote that code may have assumed that an empty list of categories would be transformed to `None` by the play framework